exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestService.java

```diff
@@ -72,7 +67,7 @@ public void afterCommit(BlockCommittedEvent event) {
   static RawTransaction constructAfterCommitTransaction(long height) {
     String payload = "Test message on height " + height;
     return RawTransaction.newBuilder()
-        .serviceId(SERVICE_ID)
+        .serviceId(TestKitWithTestArtifact.SERVICE_ID)
```

Comment:
It must use the id from the ServiceInstanceSpec supplied to the constructor with actual id and name,
          not a constant.
          Same applies to schema — it must have namespaces so that multiple instances don't
          write over the same data.